saleor-app.ts

```diff
@@ -29,11 +29,9 @@ switch (aplType) {
       throw new Error("Rest APL is not configured - missing env variables. Check saleor-app.ts");
     }
 
-    apl = new RestAPL({
-      resourceUrl: process.env.REST_APL_ENDPOINT as string,
-      headers: {
-        Authorization: `Bearer ${process.env.REST_APL_TOKEN as string}`,
-      },
+    apl = new SaleorCloudAPL({
```

Comment:
case "rest"

In the search App I've changed `rest` to `cloud`, maybe we schould have consistency here? (I can roll back to rest if you think thats the better choice)

Reply:
I'd actually go with `saleor-cloud`, its very explicit that uses who fork will not use this